luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/factories/AbridgedLogicalTableGroupFactory.java

```diff
@@ -0,0 +1,97 @@
+// Copyright 2019 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.config.luthier.factories;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.yahoo.bard.webservice.config.luthier.Factory;
+import com.yahoo.bard.webservice.config.luthier.LuthierIndustrialPark;
+import com.yahoo.bard.webservice.config.luthier.LuthierValidationUtils;
+import com.yahoo.bard.webservice.data.config.LogicalTableGroup;
+import com.yahoo.bard.webservice.data.config.names.ApiMetricName;
+import com.yahoo.bard.webservice.data.config.names.LogicalTableName;
+import com.yahoo.bard.webservice.data.dimension.Dimension;
+import com.yahoo.bard.webservice.data.metric.MetricDictionary;
+import com.yahoo.bard.webservice.data.time.DefaultTimeGrain;
+import com.yahoo.bard.webservice.data.time.ZonedTimeGrain;
+import com.yahoo.bard.webservice.table.LogicalTable;
+import com.yahoo.bard.webservice.table.PhysicalTable;
+import com.yahoo.bard.webservice.table.TableGroup;
+import com.yahoo.bard.webservice.table.TableIdentifier;
+import org.joda.time.DateTimeZone;
+
+import java.util.LinkedHashSet;
+import java.util.Locale;
+
+/**
+ * A factory that is used by default to support LogicalTableGroup.
+ */
+public class AbridgedLogicalTableGroupFactory implements Factory<LogicalTableGroup> {
+    private static final String ENTITY_TYPE = "abridged logical table group";
+
+    /**
+     * Build a LogicalTable instance.
+     *
+     * @param name  the config dictionary name (normally the apiName)
+     * @param configTable  ObjectNode that points to the value of corresponding table entry in config file
+     * @param resourceFactories  the source for locating dependent objects
+     *
+     * @return  A newly constructed config instance for the name and config provided
+     */
+    @Override
+    public LogicalTableGroup build(String name, ObjectNode configTable, LuthierIndustrialPark resourceFactories) {
+        validateFields(name, configTable);
+
+        LogicalTableGroup tableDictionary = new LogicalTableGroup();
```

Comment:
I would call this variable `tableGroup`. Just because we're effectively using a `LogicalTableDictionary` to define
our `LogicalTableGroup`, doesn't mean we need to advertise that fact in the variable name.

Reply:
That is unnecessarily confusing. if you look 20 lines below that, we will have another variable typed `TableGroup`, an in fili-core, this type is used to describe a fully configured set of physical tables. (By that I mean it contains `apiMetricName` and `dimensions` as well)
We can either rename this as `logicalTableGroup`, or, maybe just leave it as is. 
Also I suggest somehow refactor that `TableGroup` type name. It is super confusing at first.

Reply:
On a side note, the `LogicalTableGroup` is not an interface, but a concrete class at this point. 

The reason is that we wish to let `LogicalTableDictionary` to implement that eventually, but that will lead to one of the two options:

1. directly introduce `LogicalTableGroup` to fili-core/data, which is considered unstable per last talk
2. let fili-core and all other module take the luthier as a dependency, which I consider as an ugly workaround for an unnecessarily small inheritance hack.
3. (there is actually a third way I thought of:) name a concrete class `LuthierLogicalTableGroup` in luthier, and an interface `LogicalTableGroup` in luthier. let `LuthierLogicalTableGroup` extend `LogicalTableDictionary` and then implement `LogicalTableGroup`. This way we have no module level dependency and also no premature departure from master. Michael said: "no."

So for now, the `LogicalTableGroup` is just a class. We can lift it to fili-core and do the restructuring when we clear off problems in luthier.

Reply:
I suggest renaming it to `logicalTableGroup` then. And `TableGroup` is probably never going to be renamed. Pretty much everyone relies on it in their configuration, so it's not worth the pain of changingl.

Reply:
I see your point. I will make the name change in my new commit.